plugins/include/open62541/plugin/accesscontrol_default.h

```diff
@@ -31,6 +36,16 @@ UA_AccessControl_default(UA_ServerConfig *config,
                          size_t usernamePasswordLoginSize,
                          const UA_UsernamePasswordLogin *usernamePasswordLogin);
 
+UA_EXPORT UA_StatusCode
+UA_AccessControl_defaultWithLoginCallback(UA_ServerConfig *config,
```

Comment:
I would either have the loginCallback or the usernamePasswordList, but not both.
Please remove the userNamePasswordList here.

Reply:
Actually I use the usernamePasswordList to store usernames and salted password hashes.  I just change the algorithm in the callback to compare them.  Also there are some conditions (usernamePasswordLoginSize > 0) that would need extension to reflect a possible callback.

What i really dislike in the current design are the default users that are built in the code.  We got trouble during a security audit with the user1/password and user2/password1 built in your software.  By setting only the callback I do not get rid of there, although they are not used anymore.

In PR #5951 I show what API I would really like.  No build in username/password at all.  But of course that would be an API break.

Reply:
Good point.
There was a PR right now that removed the default username/pw in the default accesscontrol plugin.
Now username/pw has to be added explicitly by renewing the default AccessControl plugin instance.